storm-core/src/jvm/backtype/storm/StormSubmitter.java

```diff
@@ -187,7 +187,7 @@ public static void submitTopology(String name, Map stormConf, StormTopology topo
      * @throws AuthorizationException
      */
```

Comment:
If we now throw an IllegalArgumentException, even if it is a Runtime Exception we should document it here, and ideally describe the situation when it is thrown.  Otherwise why declare it at all?